spyder/plugins/ipythonconsole/widgets/client.py

```diff
@@ -74,20 +74,21 @@
 #-----------------------------------------------------------------------------
 # Client widget
 #-----------------------------------------------------------------------------
-class ClientWidget(QWidget, SaveHistoryMixin):
+class ClientWidget(QWidget, SaveHistoryMixin, SpyderConfigurationAccessor):
     """
     Client widget for the IPython Console
 
     This widget is necessary to handle the interaction between the
     plugin and each shell widget.
     """
 
+    sig_append_to_history_requested = Signal(str, str)
+
+    CONF_SECTION = 'inspect current object'

```

Comment:
I think this section is incorrect. Also, I think we should the shortcuts in this widget to the IPython console (right now we're using the Internal console ones, but I think that's kind of weird).